### PR TITLE
Image fixes - Monitors

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/alerts/monitors/edit-settings.md [hunk 1]
```diff
@@ -57,10 +57,11 @@ The **View in Metrics Explorer** button on the Monitors Details page allows you
...
-1. On the **Panel settings** page, click threshold <img src={useBaseUrl('/img//monitors/thresholds-icon.png')} alt="Thresholds Icon" width="40" /> icon to view the values that you have defined for the Monitor.
-1. To view the values on chart, you may need to change the window time range in the graph to some other as the default is 15 minutes. <br/><img src={useBaseUrl('/img//monitors/thresholds-graph.png')} alt="Thresholds graph" width="950" />
+1. On the Monitors Details page, navigate to the **Trigger Conditions** section and note the thresholds values defined for Critical and Warning data conditions. All other parameters will be set to default, such as the window to 15 minutes and the "at all times" box checked. <br/><img src={useBaseUrl('/img/monitors/view-in-explorer-page.png')} alt="view-in-explorer-page" width="450" />
+1. Click the **View in Metrics Explorer** <img src={useBaseUrl('/img/monitors/view-in-explorer-icon.png')} alt="view-in-explorer-icon" width="150" /> button. The Metrics Explorer page will display with the same threshold values applied to the panel and graph.
```
Comment (kimsauce) [inline docs/alerts/monitors/edit-settings.md:61]:
```suggestion
1. Click the **View in Metrics Explorer** button. The Metrics Explorer page will display with the same threshold values applied to the panel and graph.<br/><img src={useBaseUrl('img/monitors/view-in-explorer-icon.png')} alt="view-in-explorer-icon" width="150" /> 
```

File: docs/alerts/monitors/edit-settings.md [hunk 2]
```diff
@@ -57,10 +57,11 @@ The **View in Metrics Explorer** button on the Monitors Details page allows you
...
+1. On the **Panel settings** page, click threshold <img src={useBaseUrl('/img/monitors/thresholds-icon.png')} alt="thresholds-icon"
+width="40" /> icon to view the values that you have defined for the Monitor.
+1. To view the values on chart, you may need to change the window time range in the graph to some other as the default is 15 minutes. <br/><img src={useBaseUrl('/img/monitors/thresholds-graph.png')} alt="thresholds-graph" width="950" />
```
Comment (kimsauce) [inline docs/alerts/monitors/edit-settings.md:64]:
```suggestion
1. To view the values on chart, you may need to change the window time range in the graph to some other as the default is 15 minutes. <br/><img src={useBaseUrl('img/monitors/thresholds-graph.png')} alt="thresholds-graph" width="950" />
```

**Review by kimsauce — APPROVED** [review]

File: docs/alerts/monitors/edit-settings.md [hunk 3]
```diff
@@ -57,10 +57,11 @@ The **View in Metrics Explorer** button on the Monitors Details page allows you
...
-1. Click the **View in Metrics Explorer** <img src={useBaseUrl('/img//monitors/view-in-explorer-icon.png')} alt="View In Explorer" width="150" /> button. The Metrics Explorer page will display with the same threshold values applied to the panel and graph.
-1. On the **Panel settings** page, click threshold <img src={useBaseUrl('/img//monitors/thresholds-icon.png')} alt="Thresholds Icon" width="40" /> icon to view the values that you have defined for the Monitor.
-1. To view the values on chart, you may need to change the window time range in the graph to some other as the default is 15 minutes. <br/><img src={useBaseUrl('/img//monitors/thresholds-graph.png')} alt="Thresholds graph" width="950" />
+1. On the Monitors Details page, navigate to the **Trigger Conditions** section and note the thresholds values defined for Critical and Warning data conditions. All other parameters will be set to default, such as the window to 15 minutes and the "at all times" box checked. <br/><img src={useBaseUrl('/img/monitors/view-in-explorer-page.png')} alt="view-in-explorer-page" width="450" />
```
[the remark this answers is not in the document]
Comment (JainM6) [reply docs/alerts/monitors/edit-settings.md]: > * `pane`, not `page` (see the h2 header above it)
> * to fix your broken images, need to remove `/` before `img`

That is not a pane, that is a page.

[the remark this answers is not in the document]
Comment (JainM6) [reply docs/alerts/monitors/edit-settings.md]: Yes, that's correct, I thought you are talking about the Metrrics Explorer page.

[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/alerts/monitors/edit-settings.md]: Oh ok cool! Yes Metrics Explorer is a page for sure - thank you for clarifying

**Pull request merged**